invenio_assets/assets/package.json

```diff
@@ -65,7 +65,7 @@
     "webpack-dev-middleware": "^3.4.0",
     "webpack-hot-middleware": "^2.24.3",
     "webpack-livereload-plugin": "^2.2.0",
-    "webpack-manifest-plugin": "2.0.4",
+    "webpack-bundle-tracker": "^1.0.0-alpha.1",
```

Comment:
`alpha.1` 😮

Reply:
It's a really [small module](https://github.com/owais/webpack-bundle-tracker/blob/master/lib/index.js) actually, but already [used in a Django extension](https://github.com/owais/django-webpack-loader/). 

The original idea was to write it ourselves and inline it as a plugin in `wepback.config.js`...